IDEAS/Buildings/Components/Window.mo

```diff
@@ -13,8 +13,8 @@ model Window "Multipane window"
            dT_nominal=dT_nominal_a),
     QTra_design(fixed=false),
     Qgai(y=if sim.computeConservationOfEnergy then
-                                                  -(propsBus_a.surfCon.Q_flow +
-        propsBus_a.surfRad.Q_flow + propsBus_a.iSolDif.Q_flow + propsBus_a.iSolDir.Q_flow) else 0),
+                                                  -(propsBusInt.surfCon.Q_flow +
+        propsBusInt.surfRad.Q_flow + propsBusInt.iSolDif.Q_flow + propsBusInt.iSolDir.Q_flow) else 0),
```

Comment:
the new implementation indeed seems correct to me

Reply:
This lead to an error in conservation of energy as propsBusInt is an internal connector for which the sign of heat flow rates is not well defined. Using gain.propsBus_a instead solved the problem.